include/madrona/render/ecs.hpp

```diff
@@ -21,6 +21,77 @@ struct Renderable {
     Entity renderEntity;
 };
 
+
+
+
+// TODO: Make sure to move this to private headers which can be
```

Comment:
Either fix this todo or move this all under an impl namespace or something

Reply:
keeping it public for now - can be useful sometimes